### PR TITLE
ParallelDebugOutput: initialize wells with current report step number.

### PR DESCRIPTION
**Pull request by dr-robertk**
This should fix issue #521.

### CODE REVIEW

**Comment by blattms** [comment]
Wow that was fast. Thanks!
Fixes the problem as advertised.

File: opm/autodiff/ParallelDebugOutput.hpp [hunk 1]
```diff
@@ -71,7 +72,8 @@ namespace Opm
 
         // gather solution to rank 0 for EclipseWriter
         virtual bool collectToIORank( const SimulatorState& localReservoirState,
-                                      const WellState& localWellState )
+                                      const WellState& localWellState,
+                                      const int reportStep )
```
Comment (atgeirr) [inline opm/autodiff/ParallelDebugOutput.hpp]: This is unused and will cause a warning.

**Comment by atgeirr** [comment]
I will admit to having some trouble understanding the the content of this PR, not really knowing this code very well initially. Since it has been tested to solve the issue at hand I'll merge it anyway, after the minor nitpick has been addressed.

Thanks for your fix!

**Comment by dr-robertk** [comment]
Removed the warning. The fix was to initialize the WellManager every step so that all currently active Wells are present.

**Pull request merged**